requirements_dev.in

```diff
@@ -8,12 +8,13 @@ certifi==2023.7.22
 coverage==7.2.3
 esprima==4.0.1
 isort==5.12.0
-mypy==0.812
+mypy==1.14.1
```

Comment:
Think it's worth going to 1.15?

Reply:
Maybe, but let's not look into that in this PR